File: app/src/app/Models/ItemsViewModel.ts
type ArancelIdPorcentajeTuple = [number, number];
export class Item {

    item_Id: number; // Identificador del item
    cate_Id: number; // ID de la categoría del item
    item_Descripcion:string;
    item_Estado: boolean; // Estado del item
    item_Creacion: number; // Usuario que creó el registro del item
    item_FechaCreacion: string; // Fecha de creación del registro del item
    item_Modifica: number; // Usuario que modificó el registro del item
    item_FechaModifica: string; // Fecha de modificación del registro del item

}

File: api/Proyecto_BK.Common/Models/ItemsViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace sistema_aduana.Common.Models
{
    public class ItemsViewModel
    {
        public int Item_Id { get; set; }
        public string Item_Descripcion { get; set; }
        public int? Cate_Id { get; set; }
        public bool? Item_Estado { get; set; }
        public int Item_Creacion { get; set; }
        public DateTime Item_FechaCreacion { get; set; }
        public int? Item_Modifica { get; set; }
        public DateTime? Item_FechaModifica { get; set; }

    }
}
